scipy/optimize/_differentialevolution.py

```diff
@@ -181,6 +189,54 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     >>> result.x, result.fun
     (array([ 0.,  0.]), 4.4408920985006262e-16)
 
+
+    Next example demonstrates the parallelization capabilities.
+
+    >>>
+    from scipy.optimize import rosen
+    from scipy.optimize import differential_evolution as de
+    from scipy.misc import SPool
+    from scipy.misc import PPool
+    import time
+
+    def objfunc(params):
+        for it in xrange(1000000):
+            it**2
+        return rosen(params)
+
+    bounds = [(0,2), (0, 2), (0,2)]
+
+    start_time = time.time()
+    p = PPool()
+    result = de(objfunc, bounds, maxiter=10, polish=False, pool=p)
+    print("Parallel: %s seconds ---" % (time.time() - start_time))
+    print(result)
+
+    start_time = time.time()
+    result = de(objfunc, bounds, maxiter=10, polish=False)
+    print("Serial: %s seconds ---" % (time.time() - start_time))
+    print(result)
+    >>>
+    
+    Result in this case shows significant speedup:
```

Comment:
It might be better to choose a global minimum function, rather than `rosen`. Ackley might be a good one.  The examples could set a seed, so that they're both comparable.